src/api/Api.js

```diff
@@ -0,0 +1,31 @@
+import superagent from 'superagent';
+
+class Api {
```

Comment:
I just followed there the example from my online course at pluralsight. It wasn't discussed there in this regards and I took it at granted. 
How would you rate this? Just unusual, bad (practice), or acceptable?